.storybook/config.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import { configure, addDecorator } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
 import { checkA11y } from '@storybook/addon-a11y';
+import { withInfo } from '@storybook/addon-info';
+import centered from '@storybook/addon-centered';
+import '../src/tailwind.src.css';
 
+addDecorator(withInfo({
+	inline: true,
+}));
 addDecorator(checkA11y);
+addDecorator(withKnobs);
+addDecorator(centered);
```

Comment:
thinking we should use centered on a per story basis as it might interfere with the display of some stories like typography.

Reply:
Not sure if it's an error in the addon, but it only 'centers' a container. The actual components are still float left in the story.

Reply:
more worried about the width of that centered container being smaller than then ideal line length I want to show for typography, but I can deal with that when I merge the typography story in